CHANGELOG.md

```diff
@@ -1,9 +1,19 @@
 # CHANGELOG
 
+## v18.0.0.0
+
+* Upgrade to [Cumulus v18.0.0](https://github.com/nasa/cumulus/releases/tag/v18.0.0)
+* **NOTE** If you are upgrading from a version of CIRRUS earlier than `v17.0.0.3` you
+may observe a CloudFormation error.  Use the information in the `v17.0.0.3` entry below
+to resolve it
+
```

Comment:
Should we mention in the release notes that the terraform version is changing to `1.5.3`?

Reply:
@reweeden Done!